CMakeLists.txt

```diff
@@ -12,4 +12,5 @@ project(dependent_lib)
 include_directories(./)
 
 add_subdirectory(dependent)
+add_subdirectory(benchmarks)
```

Comment:
Since this is a benchmark for dependent it might be better to move it inside dependent/ folder.

Reply:
OK.